packages/python/devel/PyYAML/package.mk

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2024-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="PyYAML"
```

Comment:
I would really prefer lowecase package names.
There is a gh downlink too that is easier to craft https://github.com/yaml/pyyaml/archive/refs/tags/${PKG_VERSION}.tar.gz

Reply:
updated with the lowercase pyyaml